test/bdd/fixtures/did-trustbloc/docker-compose.yml

```diff
@@ -34,6 +34,7 @@ services:
       - DID_METHOD_HOST_URL=0.0.0.0:8090
       - DID_METHOD_TLS_CACERTS=/etc/tls/ec-cacert.pem
       - DID_METHOD_DOMAIN=${BLOC_DOMAIN}
+      - ENABLE_SIGNATURES=false
```

Comment:
@Moopli why would we want to disable signatures in our tests?

Reply:
DID-method signatures can be tested in did-method BDD tests and in edge-sandbox, rather than being tested everywhere and bloating the build time/procedure.

Reply:
Ok.